eclipse-theia/theia
# [security] perform yarn upgrade to update our dependencies

#### What it does

I noticed that a new version of `axios` is available, that fixes our only
high level vulnerability (on the dev side). Performing `yarn upgrade` picks
it up, along with a few others.

Also, pin vscode-textmate dependency to v7.0.1 in both @theia/monaco 
and @theia/plugin-ext . It turns out that more recent versions are not compatible 
with our current @theia/monaco and @theia/plugin-ext.

Fixes https://github.com/eclipse-theia/theia/issues/11772 (or at least mitigates the effects, until we update Monaco)

#### How to test

Build and run the example application. Confirm that `yarn audit` no longer reports the high-level
vulnerability about dependency `axios`

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
